### PR TITLE
Remove empty string joiners

### PR DESCRIPTION
**Pull request by squeaky-pl**
Spotted this while wandering around.
This is an artifact of running Black. These are useful to break long strings but they hinder readability when found inside the line.

### CODE REVIEW

File: inbox/providers.py [hunk 1]
```diff
@@ -87,7 +87,8 @@ def provider_info(provider_name):
                     "outlook.com.tr",
                     "outlook.com.vn",
                     "live.com",
-                    "live.com.ar" "live.com.au",
```
Comment (squeaky-pl) [inline inbox/providers.py:90]: Oops, somebody forgot a comma here.

File: inbox/api/filtering.py [hunk 2]
```diff
@@ -890,9 +890,7 @@ def page_over_shards(Model, cursor, limit, get_results=lambda q: q.all()):
                 elif "id" in last:
                     next_cursor = last["id"]
                 else:
-                    raise ValueError(
-                        "Results returned from get_query must" "have an id"
```
Comment (squeaky-pl) [inline inbox/api/filtering.py:894]: missing space

File: inbox/api/ns_api.py [hunk 3]
```diff
@@ -1992,7 +1992,7 @@ def sync_deltas():
 
     if include_types and exclude_types:
         raise InputError(
-            "Invalid Request. Cannot specify both include_types" "and exclude_types"
```
Comment (squeaky-pl) [inline inbox/api/ns_api.py:1995]: missing space

File: inbox/api/ns_api.py [hunk 4]
```diff
@@ -2140,7 +2140,7 @@ def stream_changes():
...
```
Comment (squeaky-pl) [inline inbox/api/ns_api.py:2143]: missing space

**Review by wojcikstefan — APPROVED** [review]
Very nice cleanup!

**Pull request merged**